string robot_id
---
bool success
string status_message
float32 robot_fitness